src/hooks/http.test.js

```diff
@@ -296,7 +389,39 @@ describe('http hook', () => {
     );
   });
 
-  test('getHookedClientRequestArgs', () => {
+  test('httpRequestArguments -> http(objectUrl)', () => {
+    const url = new URL('https://x.com');
+    const expected1 = {
+      url,
+      options: undefined,
+      callback: undefined,
+    };
+    expect(httpHook.httpRequestArguments([url])).toEqual(expected1);
+  });
+
+  test('httpRequestArguments -> http(objectUrl, options)', () => {
+    const url = new URL('https://x.com');
+    const options = { a: 'b' };
+    const expected1 = {
+      url,
+      options,
+      callback: undefined,
+    };
+    expect(httpHook.httpRequestArguments([url, options])).toEqual(expected1);
+  });
+
+  test('httpRequestArguments -> http(objectUrl, callback)', () => {
```

Comment:
Shouldn't there be also a test with `http(callback)`?

Reply:
Didn't find this case in the docs